src/CommonGLPI.php

```diff
@@ -955,14 +955,9 @@ public function showNavigationHeader($options = [])
         }
         $target         = $_SERVER['PHP_SELF'];
         $extraparamhtml = "";
-        $withtemplate   = "";
 
         if (is_array($options) && count($options)) {
             $cleanoptions = $options;
-            if (isset($options['withtemplate'])) {
-                $withtemplate = $options['withtemplate'];
-                unset($cleanoptions['withtemplate']);
```

Comment:
Maybe this unset should be preserved. Can probably be handled on line 963

Reply:
agree with that